target/linux/ath79/dts/qca9563_dlink_dir-859-a1.dts

```diff
@@ -1,9 +1,4 @@
-// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
-
-#include "qca956x.dtsi"
-
-#include <dt-bindings/gpio/gpio.h>
```

Comment:
GPIOs are still used in this file, please keep `#include <dt-bindings/gpio/gpio.h>`

Reply:
Is this indeed required here, or more of a stylistic issue?
I see many devices only including these in the dtsi.

Reply:
It is stylistic. We will also get it over the include, but I think it is better to include the headers for the stuff used directly.